notes/stories/literature/novel-various.st.txt

```diff
@@ -1,3 +1,73 @@
+academia [Professor Lidenbrock was a stereotypical oblivious German academic who was interested solely in advancing science by disproving the prevailing theory that the Earth had an intense source of heat at its center.]
+the scientist profession [The trained geologist Professor Lidenbrock and his nephew protégé Alex were in their element as they explored the bowels of the Earth.]
+human character stereotype [The party guide Hans was portrayed as a stoic Scandinavian to the extreme.]
+risk taking vs. playing it safe [Professor Lidenbrock single-mindedly insisted that the party take big risks at every turn in an effort to reach the center of the Earth, while his nephew Alex constantly countered him with suggestions to take the cautious route - or else return home the same way they came.]
+
+:: Minor Themes
+cryptographic cypher [Professor Lidenbrock and Alex broke Arne Saknussemm's variant on a transposition cipher to read the 16th century Icelandic alchemist's encrypted message describing how he'd made it to the center of the Earth and back.]
+romantic love [Alex was mooning over his love interest, Gretchen.]
+female stereotype [Professor Lidenbrock goddaughter Gretchen was a stereotypical timid and obedient 19th century female.]
+seasickness [Professor Lidenbrock suffered from this sickness on the voyage to Iceland to his embarrassment.]
+heresy [Arne Saknussemm was persecuted for heresy in 1573 and his books burned.]
+hunting [Hans was an eiderduck hunter by trade.]
+hospitality [A peasant family welcomed Professor Lidenbrock and his party into their home and treated them like kings.]
+human pleasure [Professor Lidenbrock and Alex waxed poetic at times about the wonders of nature they beheld while trekking through the bowels of the Earth.]
+compassion [A nearly dying of thirst Alex was touched when the otherwise oblivious and hot-tempered Professor Lidenbrock showed him a modicum of compassion by addressing him as "Poor boy".]
+solitary confinement as a punishment [A lost and alone Alex wondered to himself as to how many prisoners had gone mad from being locked up in solitary confinement.]
+human in danger [Alex was lost alone for days in the bowels of the Earth.]
+despair [Alex gave himself up for dead after wondering around alone in the bowels of the Earth for days on end, thinking he'd never find Professor Lidenbrock and Hans again.]
+facing death [Days of wonder around alone in the bowels of the Earth left Alex thinking he was goner. Professor Lidenbrock and Alex thought they might well die at one point when they were in the process of being shot up out of a volcano.]
+perilous voyage at sea [The party got caught in a severe lightning storm while traversing a vast subterranean in their makeshift raft.]
+isolated prehistoric land [The party discovered an primeval forest deep inside the Earth. In its depths they were stunned to find a prehistoric humanoid more than twelve feet in height who was watching over a herd of mastodons.]
```

Comment:
Is this not a major theme??

Reply:
While striking, it was only two or there out of 55 chapters by my count - so minor territory, I'd say.

Reply:
Couldn't you say the conclusion was that they had discovered such a thing and returned to brag about it? Your decision either way, just seems odd to me.

Reply:
Well, I'll put the onus on others to read it and say different. It wasn't much featured. The point that was most sensational is that the Earth did not have a central source of heat as the prevailing theory of the day claimed.

Reply:
I knocked up **isolated prehistoric land** to major after revisiting the text. In short, they were technically roaming through the prehistoric region from Chapter 32 through Chapter 39 - not an unsignificant chuck of the book.